cloudinit/config/schema.py

```diff
@@ -602,14 +627,44 @@ def load_doc(requested_modules: list) -> str:
 
 
 def get_schema() -> dict:
-    """Return jsonschema coalesced from all cc_* cloud-config module."""
-    full_schema = {
-        "$schema": "http://json-schema.org/draft-04/schema#",
-        "id": "cloud-config-schema",
-        "allOf": [],
-    }
+    """Return jsonschema coalesced from all cc_* cloud-config modules."""
+    paths = read_cfg_paths()
+    schema_files = glob.glob(
+        os.path.join(paths.schema_dir, "cloud-init-schema-*")
+    )
+    full_schema = None
+    if len(schema_files) > 1:
+        LOG.warning(
```

Comment:
We don't want to break the world if we don't have proper schema definitions. A warning will suffice.